test/bin/scenario.sh

```diff
@@ -1223,6 +1245,9 @@ if [ $# -lt 2 ]; then
     exit 1
 fi
 
+# Change added for USHIFT-5528
+SCENARIO_ARGS=("$@")
```

Comment:
Looking at the code, action_run already gets the command line, so we should check and assign that variable in `action_run`

Reply:
Also, let's use a concrete variable name rather than saving the whole command line. 
Something like RUN_HOST_OVERRIDE, which can be defined as empty in the beginning of the script and reset of necessary in `action_run`